# Author: Ehsan Moravveji (VSC, KU Leuven)
# Purpose: The PDM-compatible version information was missing
# Updated for 0.5.0 by Simon Branford (University of Birmingham)

diff -ruN acres-0.5.0-orig/pyproject.toml acres-0.5.0/pyproject.toml
--- acres-0.5.0-orig/pyproject.toml
+++ acres-0.5.0/pyproject.toml
@@ -1,5 +1,6 @@
 [project]
 name = "acres"
+version = "0.5.0"
 description = "Access resources on your terms"
 authors = [
   {name = "The NiPreps Developers", email = "nipreps@gmail.com"}
@@ -20,7 +21,6 @@
   "Programming Language :: Python :: 3.12",
   "Programming Language :: Python :: 3.13",
 ]
-dynamic = ["version"]
 
 [build-system]
 requires = ["pdm-backend"]
@@ -28,6 +28,8 @@
 
 [tool.pdm]
 distribution = true
+backend = 'pdm.backend'
+fallback_version = '0.5.0'
 
 [tool.pdm.build]
 source-includes = [
